overseerr-api.yml

```diff
@@ -2535,7 +2712,7 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/User'
-  /auth/login:
+  /auth/plex:
```

Comment:
I'm in support of renaming this endpoint, but be aware that it will break some third-party integrations like Organizr SSO.

Reply:
The route could become ['/auth/login','/auth/plex'] just for backwards compatibility

Reply:
Well, we are still in alpha so I think it's okay to make this change now.  We'll see what @sct says, but I think we can do away with `/auth/login`.

Reply:
+1 for changing the endpoint, makes more sense as we're adding more authentication methods

Reply:
Alright, let's leave it for now.